tic_tac_toe/game.py

```diff
@@ -24,7 +25,21 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+
+    # Make sure that...
+    #    position is a tuple
+    #    position's length is 2
+    #    every value in the tuple is an int
+    #    every int in the tuple is either 0, 1 or 2
+    # if not, return False
+
+    if not isinstance(position, tuple) \
```

Comment:
Line breakers `\` are not a good practice. If you ask me, I totally prefer to have this cleaner syntax, even we repeat the return statement.
```
if not isinstance(position, tuple):
    return False
if len(position) != 2:
    return False
if not all(isinstance(x, int) for x in position):
    return False
if any(x for x in position if not 0 <= x <= 2):
    return False
return True
```